core/CronArchive.php

```diff
@@ -462,7 +465,7 @@ public function run()
                     try {
                         if ($this->isWebsiteUsingTheTracker($idSite)) {
 
-                            if(!$this->hadWebsiteTrafficSinceMidnightInTimezone($idSite)) {
+                            if (!$this->hadWebsiteTrafficSinceLatestArchive($idSite)) {
```

Comment:
@diosmosis I don't think we can do this check. Eg just because an archiver finished last time at eg `2019-10-02 10:00:00` and there has been no visit since then, doesn't mean we shouldn't archive. 

* Eg the site has 20 segments or even when no segment then eg the day archive might have finished at `2019-10-02 09:50:00` and there could have been visits since then.
* Also we sometimes skip some archives when another archiver is currently processing the same site... like archiver launches and archives idsite=1&period=day... then another archiver launches and skips all `idsite=1` archivers because it's waiting for `period=day` to finish first. At the end the `archiving finished ts` will be updated though. I reckon the `OPTION_ARCHIVING_FINISHED_TS` is something we could almost remove as it the time is pretty much useless in terms of anything. 

Note: I may be understanding things wrong though as we know what a complicated thing it is. In general the only thing we know though is when actually requesting the archive for a specific `idsite, period, date, segment`... then we can look when this particular archive was last archived (the time the archiving started), and then we need to look if there was a visit since the archiving started

Reply:
I'm not 100% sure why I put this here. Seems like the code should work w/o it, will remove.

Reply:
I see, I was just renaming the method. There was no functional change.